ILD/compact/ILD_l5_v02/ILD_l5_o1_v02.xml

```diff
@@ -83,26 +83,7 @@
     <plugin name="InstallSurfaceManager"/>
   </plugins>
 
-  <fields>
-    <field type="solenoid" name="GlobalSolenoid" inner_field="Field_nominal_value"
-           outer_field="outerField_nominal_value" zmax="Coil_half_length"
-           inner_radius="Coil_inner_radius"
-           outer_radius="world_side" />
+  <include ref="../ILD_common_v02/Field_Solenoid_Ideal.xml"/>
+  <include ref="../ILD_common_v02/Field_FwdMagnets_Ideal_250GeV.xml"/>
```

Comment:
Shouldn't we put the 500 GeV magnet here for the incoming production ?

Reply:
A good point. For the main production with ideal fields, I would even suggest not including the fwd magnet fields at all to avoid different models for different energies.